example/vm/config.yaml

```diff
@@ -0,0 +1,128 @@
+# Date: March 22, 2024
+# Last Collector-Contrib Validation: v0.96.0
+
+# A receiver is how data gets into the OpenTelemetry Collector.
+# Generally, a receiver accepts data in a specified format,
+# translates it into the internal format and passes it to processors and exporters defined in the applicable pipelines.
+#
+# The recommended configuration is to enable the OpenTelemetry Protocol (OTLP) receiver for both gRPC and HTTP.
+# OTLP is the default protocol supported by all language implementations.
+# Depending on the environment, both grpc and http may not be necessary if the data being sent to the Collector only uses a single protocol.
+#
+# The hostmetrics receiver is OS-specific, so this configuration allows for different operating systems to be used.
+#
+# Additional receivers may be helpful to configure depending on the environment.
+
+receivers:
+    prometheus/self:
+        config:
+            scrape_configs:
+              - job_name: otel-collector
+                scrape_interval: 5s
+                static_configs:
+                - labels:
+                    collector_name: sn-collector
+                  targets:
+                    - 0.0.0.0:8888
+    hostmetrics:
+        collection_interval: 5s
+        scrapers:
+            cpu:
```

Comment:
For our Host Metrics prebuilt dashboard to work ([example](https://app.lightstep.com/jacob-demo/dashboard/host-metrics)), we want the config to enable some of the optional metrics. In particular:
* `system.cpu.utilization`
* `system.filesystem.utilization`
* `system.memory.utilization`

Also for reference, here is the hostmetrics receiver config in our Helm chart. There might be some differences to make it work in k8s though, I'm not sure about that. https://github.com/lightstep/otel-collector-charts/blob/main/charts/otel-cloud-stack/values.yaml#L90

Reply:
im not super familiar with running a collector on a VM so I think my only concern here would be the root_path.